lib/bibliothecary/parsers/clojars.rb

```diff
@@ -17,6 +17,7 @@ def self.mapping
 
       def self.parse_manifest(manifest)
         response = Typhoeus.post("#{Bibliothecary.configuration.clojars_parser_host}/project.clj", body: manifest)
+        raise Bibliothecary::ParseHostError.new("Http Error #{response.response_code} when contacting: #{Bibliothecary.configuration.clojars_parser_host}/project.clj", response.response_code) unless response.response_code == 200
```

Comment:
There's a `response.success?` method it looks like which handles all 20x instead of only exactly 200
https://www.rubydoc.info/github/typhoeus/typhoeus/Typhoeus/Response/Status#success%3F-instance_method

Reply:
Yeah, that looks like a nice improvement